pkg/api/frontendsettings.go

```diff
@@ -196,6 +196,13 @@ func (hs *HTTPServer) getFrontendSettingsMap(c *models.ReqContext) (map[string]i
 		buildstamp = 0
 	}
 
+	licensingPageReaderAccess := accesscontrol.EvalAny(
```

Comment:
As discussed with you, we could move this to the domain layer of the licensing package. To make sure rights don't diverge with ENT.

Reply:
Good call, i guess you would want to use the same permissions for the enterprise page

Reply:
Yup, I think that makes sense and I'll do it now.